### DISCUSSION TITLE
How to find buy price of an asset in output

### DISCUSSION THREAD

**Discussion by mazrrkhan** [provinzio/CoinTaxman#125] [General]
Hey Guys,

I have generated the output, having trouble finding buy price of an asset.

I only see sell rows and in the remarks i see buy, but no price at which the asset was bought

Any idea how i can see the buy price

**Comment by provinzio** [comment]
The buy part is not tax relevant per se and therefore not outputted in the export. Only buy-sell pairs are relevant. The sell and gain is exported. 

I am currently reworking the evaluation and export format. The export will become clearer and better. 

The buying/selling prices for the coins are saved to an swl database in the data folder

**Reply by mazrrkhan** [reply]
Thanks, yeah its getting a bit difficult to find the profit/loss from the report, i dont think i see the profit/loss in the csv.

**Reply by provinzio** [reply]
At the moment, only tax relevant gain is shown in the export. 

I am working on that.
